tests/api-index.test.js

```
@@ -0,0 +1,309 @@
+      res.write('partial ');
+      const err = new Error('After partial send');
+      next(err);
+    });
+    // eslint-disable-next-line no-unused-vars
+    app.use((err, req, res, _next) => {
+      const status = Number.isInteger(err?.status) ? err.status : 500;
+      const payload = { error: 'Internal server error' };
+      if (process.env.NODE_ENV !== 'production' && err?.message) {
+        payload.message = err.message;
+      }
+      try { console.error('[api] unhandled error', { msg: err?.message }); } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)